contrib/kubernetes/README.md

```diff
@@ -0,0 +1,44 @@
+# Kubernetes  Setup for DataHub
+
+## Introduction
+This directory provides the Kubernetes setup for DataHub.  This is the first version with simple YAML files.  
+The next version will contain DataHub [Helm](https://helm.sh/) chart that can be published to [Helm Hub](https://hub.helm.sh/)
+
+## Setup
+This kubernetes deployment doesn't contain the below artifacts. The idea is to use the original helm charts for deploying each of these separately.   
+
+* Kafka and Schema Registry [Chart Link](https://hub.helm.sh/charts/incubator/kafka)
+* Elasticsearch [Chart Link](https://hub.helm.sh/charts/elastic/elasticsearch)
+* Mysql [Chart Link](https://hub.helm.sh/charts/stable/mysql)
+* Neo4j [Chart Link](https://hub.helm.sh/charts/stable/neo4j)
+
+Also, these can be installed on-prem or can be leveraged as managed service on any cloud platform.
+
+## Quickstart
+1. Install Docker and Kubernetes
+2. Update the values in the configmap (datahub-configmap.yaml) with Docker hostname.  For example
+```
+ebean.datasource.host: "192.168.0.104:3306"
+ebean.datasource.url: "jdbc:mysql://192.168.0.104:3306/datahub?verifyServerCertificate=false&useSSL=true"
+kafka.bootstrap.server: "192.168.0.104:29092"
+kafka.schemaregistry.url: "http://192.168.0.104:8081"
+elasticsearch.host: "192.168.0.104"
+neo4j.uri: "bolt://192.168.0.104"
+```
+3. Create the configmap by running the following 
+```
+kubectl apply -f datahub-configmap.yaml
+```
+3. Run the below kubectl commands
+```
+kubectl apply -f datahub-secret.yaml
+kubectl apply -f datahub-gms-deployment.yaml
+kubectl apply -f datahub-frontend-deployment.yaml
+kubectl apply -f datahub-mce-consumer-deployment.yaml
+kubectl apply -f datahub-mae-consumer-deployment.yaml
```

Comment:
Isn't there an easier way of running them all at once just like we have in https://github.com/linkedin/datahub/blob/master/docker/quickstart/docker-compose.yml ?

Reply:
Yes. this can be replaced with
`kubectl apply -f datahub-*.yaml`

But this is more an intermediate step towards creating a helm chart.  After that it will be something like this
`helm install linkedin/datahub --name datahub`

Reply:
That's even better.